src/sentry/api/endpoints/team_alerts_triggered.py

```diff
@@ -49,12 +53,9 @@ def get(self, request, team):
             .annotate(count=Count("id"))
         )
 
-        counts = {str(r["bucket"].replace(tzinfo=None)): r["count"] for r in bucketed_alert_counts}
-        current_day = start.replace(
-            hour=0, minute=0, second=0, microsecond=0, tzinfo=None
-        ) + timedelta(days=1)
-        end_day = end.replace(hour=0, minute=0, second=0, microsecond=0, tzinfo=None)
-        while current_day <= end_day:
+        counts = {str(r["bucket"].date()): r["count"] for r in bucketed_alert_counts}
```

Comment:
Is there any reason we can't just use the date directly as the key here? If we do that, then it will automatically be encoded in a way that the frontend expects by our json encoder.

Reply:
now we're talking

Reply:
Based on our conversation on slack let's keep this format. Rather than coercing to a string, let's use `.isoformat()` instead just to be explicit